src/server/views/dashboard/partials/table-header-cell.njk

```diff
@@ -0,0 +1,33 @@
+{% set nameToTitleMap = {
+  type: "Type",
+  country: "Country",
+  live: "Live",
+  actionNeeded: "Action",
+  lastAnnualReviewStartDate: "Last review",
+  nextAnnualReviewStartDate: "Next review",
+  admins: "Admins",
+  listId: "Settings"
+} %}
+
+{% set orderByToAriaAttributeMap = {
+  asc: "ascending",
+  desc: "descending",
+  'none': "none"
+} %}
+
+{% macro tableHeaderCell(cell) %}
+  {% set aria = orderByToAriaAttributeMap[cell.currentlySortedBy] %}
+  <th scope="col" class="govuk-table__header govuk-table__header--dashboard dashboard-table__header" aria-sort="{{ aria }}">
+    {% if cell.name == "listId" %}
+      <span class="govuk-visually-hidden">{{ nameToTitleMap[cell.name] }}</span>
```

Comment:
Without this addition, if using your screenreader in "table" mode, and you are navigating down a single column, it will just read "Funeral directors, settings, column 8 of 8" for example. 

Or if you are sorting by next review date, and going down the column, "Translators and interpreters, next review, no review". The country is not read out, so then a user would have to navigate left to to the right column and find out what that is. That is not good UX.